lighthouse-core/audits/seo/link-text.js

```diff
@@ -18,6 +18,24 @@ const BLOCKLIST = new Set([
   'more',
   'learn more',
 ]);
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of a Lighthouse audit that provides detail on the text descriptions used for links on the page. This descriptive title is shown when all links on the page have sufficient textual descriptions. */
+  title: 'Links have descriptive text',
+  /** Title of a Lighthouse audit that provides detail on the text descriptions used for links on the page. This imperative title is shown when one or more links on the page do not have enough text describing them to be considered valid when being crawled by a search engine. */
```

Comment:
should we mention that this is basically when links have generic text? i.e "one or more links on the page do not have descriptive text and only use generic phrases like "click here"

IDK 🤷‍♂️